packages/react-component-library/src/primitives/FloatingBox/FloatingBox.tsx

```diff
@@ -49,11 +72,18 @@ export const FloatingBox: React.FC<FloatingBoxProps> = ({
     arrowElementRef,
     styles,
     attributes,
-  } = useFloatingElement(placement)
+  } = useFloatingElement(placement, undefined, targetElement)
```

Comment:
Wondering if it could work well to swap the parameters on `useFloatingElement` hence not needing to pass `undefined`.

Reply:
I did want to change the signature of `useFloatingElement` but we're exporting it for some reason – will have to leave it alone for now but we could change it for v3 or make it internal.

Reply:
I think the thinking was to expose the hook for people to use in situations where FloatingBox was not appropriate in order to try and promote consistency across downstream apps in how they approach this floating stuff. I don't think anyone is using it at the moment though..